### PR TITLE
Feature: Toasts & Snackbars

### PR DESCRIPTION
**Pull request by peec**
Demo: http://jensb.home.pkj.no/paper-toast/index.html#/toast

- Full featured toasts.
- Angular material scss only
- Actions up, data down driven
- Animations, Swipe and all event based features works.

### CODE REVIEW

**Comment by peec** [comment]
@miguelcobain This PR no longer depends on #138 and it turned out pretty nice after some refactoring and using ember-css-transitions.

**Comment by mhretab** [comment]
@peec @miguelcobain any idea when this might be merged?

**Comment by miguelcobain** [comment]
I'd like us to be able to use component blocks to specify the toast content.

File: addon/components/paper-toast.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+  _top: Ember.computed(function () { return this.get('top'); }),
+
+
+
+
+  didInsertElement() {
+    if (this.get('hide-delay')) {
+      Ember.run.later(function () {
+        this.sendAction('on-close');
+      }.bind(this), this.get('hide-delay'));
```
Comment (miguelcobain) [inline addon/components/paper-toast.js]: Use arrow functions instead of bind.

**Comment by DanChadwick** [comment]
Closing in preference to #350, which continues progress on this PR.

**Pull request closed without merging**